controllers/druid/handler.go

```diff
@@ -253,6 +253,12 @@ func deployDruidCluster(sdk client.Client, m *v1alpha1.Druid) error {
 		}
 	}
 
+	if err := deleteOrphanPVC(sdk, m); err != nil {
+		e := fmt.Errorf("Error in deleteOrphanPVC [%s]", err.Error())
+		sendEvent(sdk, m, v1.EventTypeWarning, "LIST_FAIL", e.Error())
+		logger.Error(e, e.Error(), "name", m.Name, "namespace", m.Namespace)
+	}
+
```

Comment:
- We can add a flag in case this wants does not want to be enforced ( may be user wants to keep data in pvc )

Reply:
yeah, that would make sense... default it to true though.